### PR TITLE
#752 Unified button for sharing and linking

### LINKED ISSUE #752: Make unified button for sharing and linking
Let's deliver same style to the sharing button as we have already on other buttons, with clean rectangle borders

Needs replacement on 
- profile https://nft.kodadot.xyz/rmrk/u/EJMCExEPNyuq6EofbvwD2ErZKwuZCJVTFhiDcDwuW12c187
- collection https://nft.kodadot.xyz/rmrk/collection/4c4a2f66cde689450b-CHGL100
Right now 
image

The desired look for button, recycle one from actions?
image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the KodaDot NFT gallery.
Let's do a quick check before the merge.
Your contribution will be an eternal part of the community codebase.

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting this PR, please make sure:
- [x] Code builds clean without any errors or warnings
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it on mobile and everything works
- [ ] I found edge cases

### What's new? (may be part of changelog)
- This PR closes #752

### If issue had bounty label ($,$$,$$$)
- Fill up your KSM address: 
https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU

### Community participation
- [x] Are you at KodaDot Discord? If not yet, [let's join us - https://discord.gg/35hzy2dXXh](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] Whenever my fix has changed **something** on UI, a screenshot for others, is more than welcome

Screenshot 2021-09-23 at 7 27 53 PM

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
LGTM, the only thing that bothers me is that the Sharing component is not clickable on the inside

edit: screenshot with pink clickable zone to show what i mean
Screenshot 2021-09-23 at 16-37-49 Second Rate Clowns
.

**Comment by yangwao** [comment]
I would suggest changing to the same height as its icons in this same row?  Probably could be the same height as the Gift button?

image

**Comment by yangwao** [comment]
> LGTM, the only thing that bothers me is that the Sharing component is not clickable on the inside

Yes, from a mobile perspective and mobile lighthouse report, I guess button should be clickable all surface

**Comment by prachi00** [comment]
@yangwao @roiLeo  Have done the 2 changes mentioned.

File: src/components/rmrk/Gallery/Item/Sharing.vue [hunk 1]
```diff
@@ -1,11 +1,10 @@
 <template>
   <div class="card share">
```
Comment (roiLeo) [inline src/components/rmrk/Gallery/Item/Sharing.vue:2]: card class can be saffely remove here

**Review by roiLeo — APPROVED** [review]

**Pull request merged**